Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -212,7 +212,8 @@ void MenuBar::AddFileMenu()
   file_menu->addSeparator();
 
   m_exit_action =
-      file_menu->addAction(tr("E&xit"), this, &MenuBar::Exit, QKeySequence(Qt::ALT + Qt::Key_F4));
+      file_menu->addAction(tr("E&xit"), this, &MenuBar::Exit);
+  m_exit_action->setShortcuts({ QKeySequence::Quit, QKeySequence(Qt::ALT + Qt::Key_F4) });
```

Comment:
Any reason you didn't use `QKeySequence::Quit` for this while you were at it?

Reply:
It is in use, it uses both `QKeySequence::Quit` and the existing Alt+F4 shortcut. Granted, on my system (Arch + KDE Plasma) Alt+F4 still works anyway. Unless I'm misunderstanding?

Reply:
Well, yeah, missed the word "_just_" in there. But now that you mention it, you're right; macOS uses Command+Q instead of Alt+F4 (or something like that).

Can probably leave it like that, forget that I said anything.

Reply:
macOS does indeed use Command+Q, which this adds support for. It's also common practice for Qt applications to use Ctrl+Q. I'm not sure what it's like on Windows with non-Qt applications, but on Linux a heck of a lot are starting to implement this, both Gtk and Qt alike.